internal/time.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package internal
```

Comment:
Why is the package `internal` and not `time`?

Reply:
This package doesn't seem to be very useful. Do you think removing it can reduce the code size and make the code simpler elsewhere?

Reply:
I moved validation function under the `mapcmd` package and removed custom data type about durations. [7a991c2](https://github.com/hazelcast/hazelcast-commandline-client/pull/67/commits/7a991c20b78270f67c8e763f4727f6ef1e3a3463).